src/main/java/net/dv8tion/jda/internal/requests/Requester.java

```diff
@@ -47,7 +49,7 @@
 public class Requester
 {
     public static final Logger LOG = JDALogger.getLog(Requester.class);
-    public static final String DISCORD_API_PREFIX = String.format("https://discord.com/api/v%d/", JDAInfo.DISCORD_REST_VERSION);
+    public static final String DISCORD_API_PREFIX = Helpers.format("https://discord.com/api/v%d/", JDAInfo.DISCORD_REST_VERSION);
```

Comment:
Since this seems to be the only place where this helper is used, why have this method in the first place and not apply the Locale.ROOT directly here?

Reply:
Because this PR is a draft. It’s not finished yet. The method will be used in more places where the locale might be problematic.